detect_secrets/filters/heuristic.py

```diff
@@ -68,7 +68,7 @@ def is_likely_id_string(secret: str, line: str) -> bool:
 
 @lru_cache(maxsize=1)
 def _get_id_detector_regex() -> Pattern:
-    return re.compile(r'id[^a-z0-9]', re.IGNORECASE)
+    return re.compile(r'^(id|myid|userid)|_id[^a-z0-9]', re.IGNORECASE)
```

Comment:
It looks like it doesn't enforce strings that end with `_id` (ex. my_id). It also doesn't enforce strings where id is used in its plural form (ids). Could it be worth including both?

Reply:
So this would identify the following string `my_id: POTENTIAL_SECRET` as an id_string. The idea is to identify strings that follow the format `id: POTENTIAL_SECRET`. We want to do this to filter them out as potential secrets (We detected the secret but are filtering it out of the result set since it is an id string).

Reply:
Oh okay, so it relies on the fact that all names will always be followed by `:`. What about the plural form of id? Would something like `my_ids: POTENTIAL_SECRET` be flagged?

Reply:
Right. Not necessarily only `:`. We are trying to mimic some sort of assignment operation. So it can be followed by any non-letter/non-numeric character. Ex. `space, =` etc. Currently the plural of ids wouldn't be flagged and would not be filtered out. We can add the plural incase of a series of ids possibly.